Rakefile

```diff
@@ -57,6 +57,12 @@ GEM_INFO = {
       OpenTelemetry::Exporters::Jaeger::VERSION
     }
   },
+  "opentelemetry-resource_detectors" => {
+    version_getter: ->() {
+      require './lib/opentelemetry/resource-detectors/version.rb'
```

Comment:
Should this be `resource_detectors/version.rb` or `resource/detectors/version.rb`? I'm a little confused with all the renaming below.

Reply:
Definitely the latter, fixed.